site/content/docs/5.1/utilities/text.md

```diff
@@ -103,7 +103,7 @@ Change the line height with `.lh-*` utilities.
 
 ## Line length
 
-Limit the line length with `.ll-*` utilities. It applies `max-width` rule to an element with a width based on `ch` dimension, only on block-level elements and not on inline elements.
+Limit the line length with `.ll-*` utilities. It applies `max-width` property to an element with a width based on `ch` unit. We chose this one because it is a font-based unit and because 1`ch` is the width of '0' (which is around an average character width). It requires anything but `inline` to be applied.
```

Comment:
Maybe use the passive voice and say `This one was chosen` instead of `We chose this one` (it seems to be more used in Bootstrap in the doc).

I think it is rather `1ch` (between ticks).

Reply:
Accessibility expert approved these texts